expenses.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS expenditures;
+
+CREATE TABLE expenditures (
+  id                 serial,
+  user_id            integer,
+  category_id        integer,
+  item               text,
+  transaction_date   numeric,
```

Comment:
numeric may be overkill for this, maybe something like `bigint`: https://www.postgresql.org/docs/9.1/static/datatype-numeric.html